gluten-flink/ut/src/test/java/org/apache/gluten/table/runtime/stream/custom/ScalarFunctionsTest.java

```diff
@@ -57,6 +57,23 @@ void testSubtract() {
     runAndCheck(query2, Arrays.asList("+I[0]", "+I[1]", "+I[2]"));
   }
 
+  @Test
+  void testDiv() {
+    List<Row> rows = Arrays.asList(Row.of(1, 1L, 1.0), Row.of(2, 2L, 1.0), Row.of(3, 3L, 2.0));
```

Comment:
Maybe need more cases to verify the results. Such as divide 0, divide decimal and check the precise, double divide double and check the precise same with flink.